### PR TITLE
Add '/sponge mods' command

### PR DESCRIPTION
**Pull request by Grinch**
**[SpongeCommon](https://github.com/SpongePowered/SpongeCommon/pull/1481)** | **SpongeForge**

Based on @zachoooo's work from SpongeCommon #
 
- Filters out plugins
- Ensures Minecraft, MCP, FML, MinecraftForge, SpongeAPI and SpongeForge are first in the list (none of these are filtered from this list)

**Screenshot**

Signed-off-by: Steven Downer <grinch@outlook.com>

### CODE REVIEW

File: src/main/java/org/spongepowered/mod/command/SpongeForgeCommand.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    public static CommandSpec getModsCommand() {
+        return CommandSpec.builder()
+                .description(Text.of("List currently installed mods"))
+                .permission("sponge.command.mods")
+                .arguments(optional(plugin(Text.of("mod"))))
+                .executor((src, args) -> {
+                    if (args.hasAny("mod")) {
+                        sendContainerMeta(src, args,  "mod");
+                    } else {
+                        final List<String> listStatics = Lists.newArrayList("minecraft", "mcp", "FML", "forge",
```
Comment (kashike) [inline src/main/java/org/spongepowered/mod/command/SpongeForgeCommand.java]: Can be a constant.

File: src/main/java/org/spongepowered/mod/mixin/core/common/MixinSpongeImplHooks.java [hunk 2]
```diff
@@ -483,4 +494,41 @@ public static void onCraftingRecipeRegister(CraftingRecipe recipe) {
         }
         return Optional.of(((CraftingRecipe) recipe));
     }
+
+    /**
+     * @author
```
Comment (kashike) [inline src/main/java/org/spongepowered/mod/mixin/core/common/MixinSpongeImplHooks.java]: you!

File: src/main/java/org/spongepowered/mod/mixin/core/common/MixinSpongeImplHooks.java [hunk 3]
```diff
@@ -483,4 +494,41 @@ public static void onCraftingRecipeRegister(CraftingRecipe recipe) {
...
+     */
+    @Overwrite
+    public static Text getAdditionalCommandDescriptions() {
+        return Text.of(SpongeCommand.INDENT, SpongeCommand.title("mods"), SpongeCommand.LONG_INDENT, "List currently installed mods");
+    }
+
+    /**
+     * @author
```
Comment (kashike) [inline src/main/java/org/spongepowered/mod/mixin/core/common/MixinSpongeImplHooks.java]: you!

File: src/main/java/org/spongepowered/mod/mixin/core/common/MixinSpongeImplHooks.java [hunk 4]
```diff
@@ -483,4 +494,41 @@ public static void onCraftingRecipeRegister(CraftingRecipe recipe) {
...
+     */
+    @Overwrite
+    public static void registerAdditionalCommands(ChildCommandElementExecutor flagChildren, ChildCommandElementExecutor nonFlagChildren) {
+        nonFlagChildren.register(SpongeForgeCommand.getModsCommand(), "mods");
+    }
+
+    /**
+     * @author
```
Comment (kashike) [inline src/main/java/org/spongepowered/mod/mixin/core/common/MixinSpongeImplHooks.java]: you!

**Review by kashike — CHANGES_REQUESTED** [review]

**Comment by Zidane** [comment]
Merged in https://github.com/SpongePowered/SpongeForge/commit/2c46e519e3f3583524f29947a1f2cfba74608d77

Thanks!

**Pull request closed without merging**